1.20.1/dev

desht opened the pull request:
One fix, one change which will be needed by FTB Economy

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/FTBTeam/FTB-Library-Legacy?pullRequest=88) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/FTBTeam/FTB-Library-Legacy?pullRequest=88) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/FTBTeam/FTB-Library-Legacy?pullRequest=88) it.</sub>

desht committed:
chore: added BaseMenuScreen, updated MenuScreenWrapper

Better support for any future menu-based screens

desht committed:
fix: left-clicking textfield now clears any selection

Shift-click still adjusts selection pos, as expected

https://github.com/FTBTeam/FTB-Mods-Issues/issues/1185

desht committed:
build: version -> 2001.2.2

MichaelHillcox commented:
Closing due to archived mod
